libs/blocks/table/table.js

```diff
@@ -153,6 +153,32 @@ function handleAddOnContent(table) {
   table.addEventListener('mas:resolved', debounce(() => { handleEqualHeight(table, '.row-heading'); }));
 }
 
+function setTooltipPosition(el) {
+  const tooltips = el.querySelectorAll('.milo-tooltip');
+
+  tooltips.forEach((tooltip) => {
+    const classesToCheck = ['top', 'bottom', 'left', 'right'];
+    const defaultClass = classesToCheck.find((className) => tooltip.classList.contains(className)) || 'right';
+    tooltip.setAttribute('data-tooltip-default-position', defaultClass);
+
+    tooltip.addEventListener('mouseenter', () => {
+      const isTabletOrMobile = ['TABLET', 'MOBILE'].includes(defineDeviceByScreenSize());
+      const isRtl = document.documentElement.dir === 'rtl';
+
+      if (!((el.getBoundingClientRect().width === window.innerWidth) || isTabletOrMobile)) {
+        tooltip.classList.remove(...classesToCheck);
+        tooltip.classList.add(defaultClass);
+        return;
+      }
+
+      const newClass = isTabletOrMobile === isRtl ? 'right' : 'left';
```

Comment:
Wouldn't an `&&` be better than `===` here? They're almost the same except if both are `false`. So on desktop with RTL, we'd be moving the tooltip in the opposite direction.